server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -443,8 +448,11 @@ final private[worker] class StorageManager(conf: RssConf, workerSource: Abstract
       }
     }
     if (null != diskOperators) {
+      cleanupExpiredShuffleKey(shuffleKeySet())
+      Thread.currentThread().join(RssConf.waitCleanTaskSubmitBeforeCloseTimeoutMs(conf))
+      val timeout = RssConf.workerDiskFlusherShutdownTimeoutMs(conf)
       diskOperators.asScala.foreach(entry => {
```

Comment:
I think we should use ThreadUtils.parmap to call shutdown in parallel to avoid sequential wait for timeout.